.github/workflows/main.yml

```diff
@@ -492,3 +501,113 @@ jobs:
         with:
           name: 'obs-studio-macos-${{ matrix.arch }}-notarized'
           path: '${{ github.workspace }}/${{ env.FILE_NAME }}'
+
+  macos_sparkle:
+    name: '04 - macOS Sparkle Updates'
+    runs-on: [macos-12]
+    needs: [macos_release]
+    if: needs.macos_release.outputs.run_sparkle == 'true'
```

Comment:
That does not work since it's a string, not a boolean, meaning that `"false"` would also evaluate to a truthy value. I think in the future we want to change such evaluations to `toJSON( needs.macos_release.outputs.run_sparkle)` which will evaluate `"true"` to `true`.

Reply:
Yeah using the `toJSON` pattern seems to be the suggested way to handle this, but it's effectively the same.

We could argue "why not use it already if we _add_ code" and I wouldn't have a good answer to that. 😄

Reply:
I can certainly just use `fromJSON` instead when I update this PR tomorrow.